zerver/tests/test_message_send.py

```diff
@@ -2091,7 +2092,9 @@ def test_ensure_stream_gets_called(self) -> None:
 
 class TestCrossRealmPMs(ZulipTestCase):
     def make_realm(self, domain: str) -> Realm:
-        realm = Realm.objects.create(string_id=domain, invite_required=False)
+        realm = do_create_realm(string_id=domain, name=domain)
+        realm.invite_required = False
+        realm.save(update_fields=["invite_required"])
```

Comment:
It would be cleaner to use a `do_` method for mutating the Realm's `invite_required` field as well.

Reply:
(Or possibly we should just make that an optional parameter, since a lot of these tests seem to want to reset it)